### ISSUE TITLE
RipMe no longer ripping Imgur images or folders, when ripping Reddit user accounts

### ISSUE THREAD

**Issue by onesimpleclik** [RipMeApp/ripme#1456]
* Ripme version: 1.7.88
* Java version: 
* Operating system: Windows 10

* Exact URL you were trying to rip when the problem occurred:
* Please include any additional information about how to reproduce the problem:

## Expected Behavior

Normally RipMe will scrape the Reddit users comments & posts for any images that are hosted on Reddit itself, or direct Imgur links, or Imgur folders.

## Actual Behavior

When ripping images from reddit accounts, from posts/comments made by the user, RipMe no longer detects & downloads images that are hosted on Imgur (either as a direct link, or images within a imgur folder). RipMe will only download images hosted on Reddit itself.

Only images that fall under https://i.redd.it/filename.filetype are being captured, not https://i.imgur.com/filename.filetype or https://imgur.com/a/foldername

**Comment by mkolakowski** [comment]
This could be due to imgur now requiring users to sign in to view mature content

![image](https://user-images.githubusercontent.com/31713098/66875449-55719100-ef7c-11e9-8de9-1d660bb3391f.png)

**Comment by flerpmcnasty** [comment]
I believe the required login for mature content change recently implemented by imgur as detailed at https://blog.imgur.com/2019/10/14/discontinuing-support-for-nsfw-reddit-subsections/ is the problem.

Running from CLI I get the following type of response and it fails to download:

    Retrieving https://imgur.com/a/yLAckpV/all
[!] Falling back to /noscript method
    Retrieving https://imgur.com/a/yLAckpV/noscript

If you try loading either of those without being logged in you get a "sign in required" as mentioned above by @mkolakowski 

Seems like an authentication method for imgur is necessary.

**Comment by onesimpleclik** [comment]
That sounds like a pain. Is it even possible to add login/authentication to RipMe?

**Comment by cyian-1756** [comment]
> Is it even possible to add login/authentication to RipMe?

Yup. Luckily all you need is a client-id to access nfsw albums and imgur exposes those in javascript files

**Comment by cyian-1756** [comment]
Is this still an issue with ripme 1.7.90?
